src/Core/Models/Domain/LoginUri.cs

```diff
@@ -18,11 +20,18 @@ public class LoginUri : Domain
         public LoginUri(LoginUriData obj, bool alreadyEncrypted = false)
         {
             Match = obj.Match;
+
+            if (obj.UriChecksum != null)
+            {
+                UriChecksum = new EncString(obj.UriChecksum);
+            }
```

Comment:
⚠️  This is not needed, as it's handled by the `BuildDomainModel(..)` next line.
You should just add `nameof(UriChecksum)` to the `_map` HashSet above and it'll take care of creating the `EncString` for you. Also it'll handle building the data model in the `ToLoginUriData` method.